spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultSchedulerService.java

```diff
@@ -70,6 +70,8 @@
 	private final static String DEPLOYER_PREFIX = "deployer.";
 	private final static String COMMAND_ARGUMENT_PREFIX = "cmdarg.";
 	private final static String DATA_FLOW_URI_KEY = "spring.cloud.dataflow.client.serverUri";
+	private final static String KUBERNETES = "kubernetes";
```

Comment:
We could perhaps move the PLATFORM_TYPE constant values (kubernetes, cloudfoundry and local) from TaskPlatformFactory classes into some common or core module and that way we can re-use them in other places.

Reply:
I love your idea!   But what I will do is to pull the types that are enumerated in each of the impl's of `TaskPlatformFactory` to the `TaskPlatformFactory` itself.  This way we have a single location from which to pull the types.   I'll submit this change in the update to the PR.

Reply:
Tell me what you think.

Reply:
This is a nice approach! :-)